_integration-test/run.sh

```diff
@@ -146,7 +147,7 @@ for service in $(docker compose ps --format json | jq -r '.[]| select(.State !=
 done
 
 if [[ $service_failures ]]; then
-  $dc ps
+  echo "$SERVICES_NOT_RUNNING"
```

Comment:
Maybe a little more jq on this bad boy? We really on care about name and status, also don't care about geoipupdate.

Reply:
Fair point! Will do.